core/includes/file.inc

```diff
@@ -543,19 +543,23 @@ SetHandler Drupal_Security_Do_Not_Remove_See_SA_2006_006
 <IfModule mod_php5.c>
   php_flag engine off
 </IfModule>
+
 EOF;
 
   if ($private) {
-    $lines = <<<EOF
-    # Deny all requests from Apache 2.4+.
-    <IfModule mod_authz_core.c>
-      Require all denied
-    </IfModule>
-
-    # Deny all requests from Apache 2.0-2.2.
-    <IfModule !mod_authz_core.c>
-      Deny from all
-    </IfModule>
+    $lines .= <<<EOF
+
+# Deny all requests from Apache 2.4+.
+<IfModule mod_authz_core.c>
+  Require all denied
+</IfModule>
+
+# Deny all requests from Apache 2.0-2.2.
+<IfModule !mod_authz_core.c>
+  Deny from all
+</IfModule>
+
+
```

Comment:
Hm, I think at the time I had thought that if something else appends to the file it should start with an extra line to create a gap between the original and the new code. However, that seems unlikely and it does seem a bit strange. I'll nix it.